pallets/dapp-staking-v3/src/types.rs

```diff
@@ -1597,7 +1608,7 @@ pub struct TiersConfiguration<NT: Get<u32>, T: TierSlotsFunc, P: Get<FixedU128>>
     pub reward_portion: BoundedVec<Permill, NT>,
     /// Requirements for entry into each tier.
     /// First entry refers to the first tier, and so on.
-    pub tier_thresholds: BoundedVec<TierThreshold, NT>,
+    pub tier_threshold_values: BoundedVec<Balance, NT>,
```

Comment:
You can keep the same name, no need to change it.

It will be less confusing for the UI team as well.